tests/manipulator_test.py

```diff
@@ -0,0 +1,321 @@
+import unittest
```

Comment:
You would usually start the name of test files with `test_`, so name it is `test_manipulator.py`.  Some test runnings search for filename starting with `test`.